versions/v6/templates/story/related-stories.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+$related_stories = related_stories_get_stories( $post->ID );
+
+if ( $related_stories ) :
+?>
+<aside>
```

Comment:
Since `<aside>` is a landmark element, we're probably going to need to add a unique label for it--this can be done by adding a unique ID to the "Related Stories" h2, and adding `aria-labelledby="..."` to the `<aside>`